pkg/status/taskrunpod.go

```diff
@@ -23,7 +23,7 @@ import (
 
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
 	listers "github.com/tektoncd/pipeline/pkg/client/listers/pipeline/v1alpha1"
-	"github.com/tektoncd/pipeline/pkg/reconciler/taskrun/resources"
+	podconvert "github.com/tektoncd/pipeline/pkg/pod"
```

Comment:
Any reason for naming it `podconvert` instead of pod ?

Reply:
There's a variable called `pod` below that causes problems. When this is all done I hope to avoid this named import.